ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -10,6 +10,9 @@ ActingAppType,string,Optional,AuditEvent,Enumerated,Process|Service|Resource|URL
 ActingAppType,string,Optional,Authentication,Enumerated,Process|Service|Resource|URL|SaaS application|CSP|Other,
 ActingAppType,string,Optional,FileEvent,Enumerated,Process|Service|Resource|URL|SaaS application|CSP|Other,
 ActingAppType,string,Optional,UserManagement,Enumerated,Process|Service|Resource|URL|SaaS application|CSP|Other,
+ActingOriginalAppType,string,Optional,AuditEvent,,,
+ActingOriginalAppType,string,Optional,UserManagement,,,
```

Comment:
@vakohl - As discussed, please generate AsimTester.csv through script by reading the original schemas and dependencies.

Reply:
@anki-narravula done. Shared results with you.